docs/user_guides/single_table/gaussian_copula.rst

```diff
@@ -351,73 +350,6 @@ Now that we have discovered the basics, let's go over a few more
 advanced usage examples and see the different arguments that we can pass
 to our ``GaussianCopula`` Model in order to customize it to our needs.
 
-How to set transforms to use?
```

Comment:
After speaking with @npatki we decided to just remove this section as the main reason it was here was so that people would know how to replace `one-hot-encoding` with something faster. It also doesn't exist in any of the other models